src/hotspot/cpu/x86/stubGenerator_x86_64_aes.cpp

```diff
@@ -3527,6 +3527,8 @@ void StubGenerator::aesgcm_avx512(Register in, Register len, Register ct, Regist
 
   __ bind(MESG_BELOW_32_BLKS);
   __ subl(len, 16 * 16);
+  __ cmpl(len, 256);
```

Comment:
From the stylistic logic, this should be written as `16 * 16`, to match the surrounding `subl` and `addl`.